br/pkg/restore/restorer.go

```diff
@@ -226,6 +226,9 @@ type MultiTablesRestorer struct {
 	workerPool       *util.WorkerPool
 	fileImporter     BalancedFileImporter
 	checkpointRunner *checkpoint.CheckpointRunner[checkpoint.RestoreKeyType, checkpoint.RestoreValueType]
+
+	fileCount int
```

Comment:
need lock here or make it atomic?

Reply:
The variable `fileCount` is modified in `GoRestore` and load in `WaitUntilFinish`(Notice that `fileCount` is not used in workerpool or any other goroutines). Actually, the two functions are always appear together like this:
```
err := restorer.GoRestore()
if err != nil {
  return err
}
err = restorer.WaitUntilFinish()
if err != nil {
  return err
}
```